homeassistant/components/homematicip_cloud/helpers.py

```diff
@@ -0,0 +1,9 @@
+"""Helper functions for Homematicip Cloud Integration."""
+
+
+def is_error_response(response) -> bool:
+    """Response from async call contains errors or not."""
+    if isinstance(response, dict):
+        return "errorCode" in response and response["errorCode"] != ""
```

Comment:
This can be simplified:

```suggestion
        return response.get("errorCode") not in ("", None)
```